rust-bins/docs/keygen.md

```diff
@@ -20,11 +21,14 @@ Generate keys for an anonymity revoker. The following options are supported
   anonymity revokers. It has meaning in cryptographic protocols (it is used as the evaluation point of the sharing polynomial) and so it must be non-zero.
 - `--description` string, a free-form description containing information that users will want to see about the anonymity revoker
 - `--global` a filename, containing cryptographic parameters of the chain anonymity revokers will be a part of.
+- `--in-len` an integer determining how many words to user is asked to enter.
 - `--name` a string, a simple identifier of the anonymity revoker. Chosen by themselves.
 - `--url` a URL with the website of the anonymity revoker
-- `--rand-input`, a filename with the randomness input. It must contain at least 64 bytes.
 - `--out`, a filename where the private keys will be emitted
 - `--out-pub`, a filename where the public data will be emitted. This is the data that must go to the chain.
+- `--recover-from-phrase` if set, recover keys from backup phrase. Otherwise, fresh keys are generated.
+- `--no-confirmation` if set, do not ask user to re-enter generated recovery phrase.
+- `--no-verification` if set, do not verify the validity of the input. Otherwise the input is verified to be a valid BIP39 sentence.
```

Comment:
Why is this flag necessary?

Is it just generality for its own sake or does it have utility? It complicates the logic because there are  now 2 versions of many things so I'd prefer to remove the not-verified option and just require that everything is as in the spec.

Reply:
I found it useful for testing, because you can just enter 24 times nothing with --no-verification. It also allows people to enter their own randomness if they don't have a ledger but something else. But it's probably not needed for AR use case with the ledger.

Reply:
Ok.